mgmt/RecordsConfig.cc

```diff
@@ -55,8 +55,6 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.proxy_binary", RECD_STRING, "traffic_server", RECU_NULL, RR_REQUIRED, RECC_NULL, nullptr, RECA_NULL}
```

Comment:
This won't work anymore. It was used by TM.

Reply:
So, KIWF.